app/views/users/show.html.haml

```diff
@@ -1,14 +1,14 @@
 %header.entry-header
   - user_name = @user.full_name
-  %h1{ class: with_admin_css_class_if_needed(@current_user, ['entry-title']) }
-    = user_name.empty? ? @user.email : user_name
-    - if @user.admin?
-      %span.small
-        (#{ t('.is_an_admin') })
-
-    - if @current_user.admin?
-      = link_to edit_icon, admin_only_edit_user_account_path(@user), class: ['shf-icon', 'edit-user-account-icon'], title: t('.edit_user_account', name: user_name)
-      = link_to user_profile_icon, admin_only_user_profile_edit_path(@user), class: ['shf-icon', 'edit-user-profile-icon'], title: t('.edit_user_profile', name: user_name)
+  - title_escaped = h(user_name.empty? ? @user.email : user_name)
+  - title_html_safe = ''
+  - if @user.admin?
+    - title_html_safe << tag.span(" (#{ t('.is_an_admin') })", class: 'small')
+  - if @current_user.admin?
+    - title_html_safe << ' ' + link_to(edit_icon, admin_only_edit_user_account_path(@user), class: ['shf-icon', 'edit-user-account-icon'], title: h(t('.edit_user_account', name: user_name)))
+    - title_html_safe << ' ' + link_to(user_profile_icon, admin_only_user_profile_edit_path(@user), class: ['shf-icon', 'edit-user-profile-icon'], title: h(t('.edit_user_profile', name: user_name)))
+  - title = title_escaped.html_safe + title_html_safe.html_safe
+  = content_title(title, user: @current_user)
```

Comment:
`title_html_safe`:  Ditto my comment for `app/views/user_profiles/_edit.html.haml`:

The goal is to have line 11 (where info is passed to the content_title method) clearly describe the information that is going to be used to construct the `content_title`.  I'd suggest using a few descriptive variables so that the individual pieces are really clear.  Maybe you end up with something like:
```
- title = user_name_for_display(@user) + show_if_user_is_admin(@user) + edit_acct_and_profile_links 
```
(This would go under the if @current_user.admin? clause.   `title` would be something else by default.)